plenario/api/point.py

```diff
@@ -267,7 +267,7 @@ def _detail_aggregate(args):
         resp.headers['Content-Type'] = 'application/json'
 
     elif datatype == 'csv':
-        resp = make_csv(time_counts)
+        resp = form_csv_detail_response(['point_date', 'hash'], time_counts)
         resp.headers['Content-Type'] = 'text/csv'
```

Comment:
I think this has been around for a while, but was never covered with a test. make_csv would be provided with a list of dictionaries that, without some processing, it would not be able write to a file. 

If Python did manage to make it past that line, it would have most definitely crashed on the resp.headers line.
make_csv does not return a response object that can be accessed in that way.